filebeat/prospector/prospector_log.go

```diff
@@ -112,15 +116,37 @@ func (l *Log) Run() {
 	}
 }
 
+// glob detects the use of "**" and expands it to standard glob patterns up to a max depth
+func glob(g string) ([]string, error) {
+	var globs []string
+	if filepath.Base(g) == "**" {
```

Comment:
Does this mean it only detects `**` at the end of the path? Does `/foo/**/bar` work?